Makefile

```
@@ -52,6 +52,7 @@ export SELF_SIGNED_CERTS   ?= true
 # Setting the INSTALLATION_TYPE to managed-api will configure the values required for RHOAM installs
 export INSTALLATION_TYPE   ?= managed
 
+export ALERT_SMTP_FROM ?=noreply-alert@devshift.org
```

```suggestion
export ALERT_SMTP_FROM ?= noreply-alert@devshift.org
```
Seems like it's passing the formatting anyway but maybe if we want to keep it in line with other envs?

---

Done, but just saying it would nicer if you had pointed that out in the last comment. Now the builds have been retriggered :(

---

Agree, didn't notice it at the first time